docs/product-assembly_dm-on-openshift/src/main/asciidoc/imagestreams-file-install-proc.adoc

```diff
@@ -1,20 +1,36 @@
 [id='imagestreams-file-install-proc']
 = Ensuring the availability of image streams 
 
-You must ensure that the image streams that are required for the deployment are available in your OpenShift environment. Some versions of the OpenShift environment include the necessary image streams. You must check if they are available. If they are not available, you must download and install the `rhdm70-image-streams.yaml` file.
+You must ensure that the image streams that are required for the deployment are available in your OpenShift environment. Some versions of the OpenShift environment include the necessary image streams. You must check if they are available. If they are not available, you must download and install the
+ifdef::DM[`rhdm70-image-streams.yaml`]
+ifdef::PAM[`rhpam70-image-streams.yaml`]
+file.
 
 .Procedure
 . Run the following commands:
 +
```

Comment:
Not sure what's different here, but the plus sign is being rendered in this case.